### ISSUE TITLE
Support node v20

### ISSUE THREAD

**Issue by robert-dean** [epaew/eslint-plugin-filenames-simple#636]
## Describe the bug

```
error eslint-plugin-filenames-simple@0.8.0: The engine "node" is incompatible with this module. Expected version "^14.17.0 || ^16.0.0 || ^18.0.0". Got "20.0.0"
error Found incompatible module.
```

## Expected behavior

Node version 20 to be supported

## To reproduce

Try to add to any node v20 package

### Versions

- Node.js: 20
- ESLint: 8.40.0
- plugin: latest

**Comment by robert-dean** [comment]
@epaew I would be happy to contribute if you're open to it

**Comment by chandrashekhar07** [comment]
@epaew Do you have any plan to upgrade to node v20?

**Comment by epaew** [comment]
Resolved in #675 and I published v0.9.0.
